recipes/kim_edn/meta.yaml

```diff
@@ -10,9 +10,9 @@ source:
   sha256: bab88a0a890598577a5dabe0ff19ed573ef7a0be547e5a47b15deeb4ec711db5
 
 build:
+  noarch: python >=3.6
```

Comment:
Just `noarch: python` please.

Reply:
Again the appveyor fails, but if you look at the failure message it does not have anything to do with my package.

Reply:
We are just cleaning the recipe now and making it according to the conda-forge standards. `noarch` recipes are built only on Linux and save lots of CI time. 

The AppVeyor errors are common and not related to your package. For `noarch` recipes we just want the Linux builds to pass, for non `noarch` recipes, only one Windows build is ok, so we can ignore the AppVeyor errors.